sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,444 @@
+-- Query for firefox_ios_derived.feature_usage_metrics_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE start_date DATE DEFAULT "2021-01-01";
+
+DECLARE end_date DATE DEFAULT current_date;
+
+WITH dau_segments AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS dau
+  FROM
+    `mozdata.firefox_ios.metrics`
+  WHERE
+    DATE(submission_timestamp) >= start_date
+  GROUP BY
+    1
```

Comment:
Please use field names instead of numbers in GROUP BY

Reply:
As per our agreed SQL Style, we prefer using explicit field names in the GROUP BY section of the query (with specific exceptions to this rule):
https://docs.telemetry.mozilla.org/concepts/sql_style.html#be-explicit